client/homebrew/pages/basePages/listPage/listPage.jsx

```diff
@@ -183,9 +183,18 @@ const ListPage = createClass({
 
 	getSortedBrews : function(brews){
 		const testString = _.deburr(this.state.filterString).toLowerCase();
+
```

Comment:
This works, but the triple-nesting of functions seems more complicated than necessary.

How about this:

```
	getSortedBrews : function(brews){
		const testString = _.deburr(this.state.filterString).toLowerCase();

		brews = _.filter(brews, (brew)=>{
			const brewStrings = _.deburr([
				brew.title,
				brew.description,
				brew.tags].join('\n')
				.toLowerCase());

			return brewStrings.includes(testString);
		});
		return _.orderBy(brews, (brew)=>{ return this.sortBrewOrder(brew); }, this.state.sortDir);
	}
```

Reply:
Looks good to me! I've pushed that through in the latest commit.